daml-script/dump/src/main/scala/com/daml/script/dump/Config.scala

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.script.dump
+
+import java.nio.file.{Path}
+import java.io.File
+
+final case class Config(
+    ledgerHost: String,
+    ledgerPort: Int,
+    parties: List[String],
+    outputPath: Path,
+    sdkVersion: String,
+    damlScriptLib: String,
+)
+
+object Config {
+  def parse(args: Array[String]): Option[Config] =
+    parser.parse(args, Empty)
+
+  private val parser = new scopt.OptionParser[Config]("script-dump") {
```

Comment:
Any reason to add a sub-command in case you need to support some sort of alternative action in the future?

Reply:
oh sorry forgot to respond to this before merging. Yeah we might want to change this. I’d rather revisit the whole CLI once ti becomes stable or at least released than try to predict what we might want to add.